app/controllers/users_controller.rb

```diff
@@ -1,7 +1,8 @@
 # frozen_string_literal: true
 # Users Controller class
 class UsersController < ApplicationController
-  before_action :set_user, only: %i(show edit update)
+  prepend_before_action :set_user, only: %i(show edit update edit_intent
```

Comment:
Don't we need to make this change in all of our other controllers as well? I'll open up a new issue.